drivers/filesystems/udfs/CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ if(MSVC)
 endif()
 
 if(CMAKE_C_COMPILER_ID STREQUAL "GNU")
-    target_compile_options(udfs PRIVATE -Wno-unused-but-set-variable)
+    target_compile_options(udfs PRIVATE -Wno-unused-but-set-variable -Wno-stringop-overflow)
```

Comment:
Why for this PR ? Maybe we should fix them.

Reply:
Oh, that's the UDF driver, I don't really want to look into it :D

Reply:
But that's a general problem - when various optimization levels are enabled, different errors pop up here and there